app/models/course.rb

```diff
@@ -144,11 +144,11 @@ def new_draft_attributes
           class_name: "CourseEnrichment"
 
   scope :within, ->(range, origin:) do
-    joins(:sites).merge(Site.within(range, origin: origin))
+    joins(site_statuses: :site).merge(SiteStatus.where(status: :running)).merge(Site.within(range, origin: origin))
   end
```

Comment:
This looks like its related to 
![image](https://user-images.githubusercontent.com/470137/135344957-fa933dad-c086-4e83-a31c-a76a6146d60a.png)


Whereby range is `50`